### PR TITLE
more pivot errors fixed

### PR DESCRIPTION
**Pull request by ehwenk**
more fixes to pivoting problems:
- minor changes copied from traits.build back to austraits.build process.R file
- had to add locations to Yang_2023
- Witkowski_1991 would only recognised plant ages when I wrote them in letters; tried many many other tricks

### CODE REVIEW

File: data/Witkowski_1991/metadata.yml [hunk 1]
```diff
@@ -70,8 +70,8 @@ contexts:
   category: entity_context
   var_in: plant_age
   values:
-  - value: 10
-  - value: 21
+  - value: ten
```
Comment (yangsophieee) [inline data/Witkowski_1991/metadata.yml:73]: I wonder if it's an issue with converting to character again. Did you try '10' and '21'?

**Review by yangsophieee — APPROVED** [review]
Really good work fixing all these pivot issues..

File: data/Witkowski_1991/metadata.yml (hunk 1 above)
Comment (ehwenk) [reply data/Witkowski_1991/metadata.yml:73]: Yes, I tried '10' and '21'; I checked `metadata_check_custom_R_code("Witkowski_1991)` to see how it was read in; I tried retyping the cells; I tried assigning it to a different context category; I tried adding "mutate to as.character" through custom_R_code and nothing worked. All rows in the column have a value, so it isn't being read in strangely. I'd much rather not use this trick, but also needed to move on. @yangsophieee

Comment (yangsophieee) [reply data/Witkowski_1991/metadata.yml:73]: That's so weird!!! And yes I think we should move on.

**Pull request merged**